frontend/src/js/components/app.js

```diff
@@ -119,7 +122,7 @@ export const AppRoot = () => {
         const keyOnlyFilters = filters.split('&').reduce((accu, item) => `${accu}:${item.split('=')[0]}&`, ''); // assume the keys to filter by are not as revealing as the values things are filtered by
         page = `${page.substring(0, splitter)}?${keyOnlyFilters.substring(0, keyOnlyFilters.length - 1)}`; // cut off the last & of the reduced filters string
       } else if (page.startsWith(activationPath)) {
-        dispatch(setAccountActivationCode(page.substring(activationPath.length + 1)));
+        dispatch(receivedActivationCode(page.substring(activationPath.length + 1)));
```

Comment:
For now I tried to stick to the existing action names for thunks and changed the action type identifier constants to camel case... where the thunk names were overlapping with the actions I tried to stick to the _describing events that occured_ from [here](https://redux.js.org/style-guide/#model-actions-as-events-not-setters) - but I surely didn't get all of them, the activation code being one of the instances 🙈. Please let me know if you come across any other poorly named actions!
To ease the distinction in (purely state changing) actions and thunks I think it might be best to unify the references to thunks across the codebase (as in renaming the actions files & references)...